app/controllers/foods_controller.rb

```diff
@@ -0,0 +1,77 @@
+class FoodsController < ApplicationController
+  before_action :authenticate_user!
+  before_action :set_food, only: %i[show edit update destroy]
+
+  # GET /foods or /foods.json
+  def index
+    @foods = Food.all
+  end
+
+  # GET /foods/1 or /foods/1.json
+  def show; end
+
+  def shooping_list
+    @foods = Food.all.where(user: current_user)
+    render :shooping_list
+  end
```

Comment:
- [ ] I noticed that your implementation of the shopping list has not been done in accordance with the project requirements. You are simply displaying all the foods in your database. Kindly see the attached below 👇 

![image](https://user-images.githubusercontent.com/46839548/234908739-7ac1065a-ee9f-458e-a13f-1d736d84ae17.png)